hooks/build

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
No need to use bash here; the contents here can be easily be POSIX compatible (and thus not only more portable but also faster on systems such as the Debian-based ones that use smaller and faster shells than bash as /bin/sh):

```sh
#!/bin/sh -e

if [ "$DOCKER_TAG" = latest -o "$SOURCE_BRANCH" = new-hub-build ]
then
    docker build --build-arg BUILDX_VERSION=latest -f $DOCKERFILE_PATH -t $IMAGE_NAME .
else
    docker build --build-arg BUILDX_VERSION=$DOCKER_TAG -f $DOCKERFILE_PATH -t $IMAGE_NAME .
fi

Reply:
Sure! Makes sense

Would you like to add a commit with updates? 😅

Reply:
Ill make the updates now